mypy_django_plugin/transformers/models.py

```diff
@@ -345,15 +345,19 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
                     related_manager_info = self.lookup_typeinfo_or_incomplete_defn_error(
                         fullnames.RELATED_MANAGER_CLASS
                     )  # noqa: E501
-                    default_manager = related_model_info.names.get("_default_manager")
```

Comment:
I guess the reason we raise an exception here if it's not found is that the code is supposed to run multiple iterations over a project to actually finish?

Reply:
But it never runs again.